### DISCUSSION TITLE
Is there anyway to reset boundingbox after rotate?

### DISCUSSION THREAD

**Discussion by ouyoungs** [fabricjs/fabric.js#10352] [Q&A]
something like that

![Snipaste_2024-12-23_16-06-46](https://github.com/user-attachments/assets/e59f31de-aa9a-4cc9-9d1d-4550a625d211)

**Comment by asturur** [comment]
It depends on the shape.
You can do that with Polygon and Path and nothing else i think.
With Path and Polyon you can do that absorbing the rotation or scaling at the end of transform into the point or segments of the path, so that the newly resulting shape looks identical but is not rotate anymore.

Alternatively if you are not using text or you don't mind complications you can put every shape inside its own group and then you can do that with groups by rotating the object inside the group and setting the group angle back to 0

**Reply by ouyoungs** [reply]
group looks like a good idea
